templates/tower-project.yaml

```diff
@@ -62,13 +62,6 @@ Parameters:
       - GLACIER
     Default: INTELLIGENT_TIERING
 
-  ScratchLifecycleExpiration:
```

Comment:
I just realized that we need to make sure that there are no Tower project stacks that are using this parameter. Could you search for `ScratchLifecycleExpiration` in the `config/` directory and delete any lines that come up?

Reply:
After doing the search I did not find any such lines in `config/`